spec/factories/sales_log.rb

```diff
@@ -39,6 +39,8 @@
       ecstat2 { "1" }
       hholdcount { "1" }
       wheel { 1 }
+      age3_known { 0 }
```

Comment:
This looks like our factory output is invalid by default? (age is not known, but also 40)

Reply:
I don't think so  - 0 is "Yes" for age3_known (following a pattern used elsewhere) which is confusing tbf

Reply:
Ah, okay. Weird, but sure.